Changes

```diff
@@ -382,8 +382,9 @@ Working version
 - #11931: Fix tricky typing bug with type substitutions
   (Stephen Dolan, review by Leo White and Jacques Garrigue)
 
-- #12037: get_extern_state potential NULL dereference.
-  (Alexander Skvortsov, report by Török Edwin,
+- #12037, #12171, #?????: Fix get_extern_state and get_intern_state
```

Comment:
Best to add a new Changes entry rather than changing the old one. Mixes up attributions unnecessarily.

Reply:
Right.